src/Features/LanguageServer/Protocol/Features/UnifiedSuggestions/UnifiedSuggestedActionsSource.CodeFixesComputer.cs

```diff
@@ -0,0 +1,403 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+using static Microsoft.CodeAnalysis.CodeActions.CodeAction;
+using CodeFixGroupKey = System.Tuple<Microsoft.CodeAnalysis.Diagnostics.DiagnosticData, Microsoft.CodeAnalysis.CodeActions.CodeActionPriority, Microsoft.CodeAnalysis.CodeActions.CodeActionPriority?>;
+
+namespace Microsoft.CodeAnalysis.UnifiedSuggestions
+{
+    internal partial class UnifiedSuggestedActionsSource
+    {
+        /// <summary>
+        /// Gets, filters, and orders code fixes.
+        /// </summary>
+        private static class CodeFixesComputer
```

Comment:
This file has all existing code just moved to a new CodeFixes-specific nested static type to avoid name clashes between types named similarly for fix all for code fixes and refactorings.
Existing code: https://github.com/dotnet/roslyn/blob/ae66fbc92c8b717eb6a9eff5bb7b3965694e50e4/src/Features/LanguageServer/Protocol/Features/UnifiedSuggestions/UnifiedSuggestedActionsSource.cs#L30-L403